docs/containers-storage.conf.5.md

```diff
@@ -167,6 +167,13 @@ The `storage.options.overlay` table supports the following options:
 **ignore_chown_errors** = "false"
   ignore_chown_errors can be set to allow a non privileged user running with a  single UID within a user namespace to run containers. The user can pull and use any image even those with multiple uids.  Note multiple UIDs will be squashed down to the default uid in the container.  These images will have no separation between the users in the container. (default: false)
 
+**force_mask** = "0000|native|shared|private"
```

Comment:
Need to mention here that it requires mount_program.

Reply:
Update man page, then LGTM

Reply:
fixed.  Should we mark it as experimental for now so we keep some freedom to change it if we need?

Reply:
SGTM